### PR TITLE
solution

### PR DESCRIPTION
**Pull request by liliya-dev**
- [DEMO LINK](https://liliya-dev.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://liliya-dev.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,18 @@
...
+      <img src="images/logo.png" alt="">
+      <nav>
+        <a class="is-active" href = "#">APPLE</a>
+        <a href = "#">SAMSUNG</a>
+        <a href = "#">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a data-qa="hover" href = "#">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+        <a href = "#">ГАДЖЕТЫ</a>
+        <a href = "#">ПЛАНШЕТЫ</a>
+        <a href = "#">ФОТО</a>
+        <a href = "#">ВИДЕО</a>
```
Comment (irenhh) [inline src/index.html:22]: menu items are usually  list items with ul container

Comment (irenhh) [reply src/index.html:22]: add uppercase styles in css, and here just usual text with the first upper case letter and lowercase others

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+body {
+  margin: 0;
+  font-family: Roboto;
+}
+
+header {
+
```
Comment (irenhh) [inline src/style.css:9]: remove this

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  flex-basis: 1024px;
+  flex-shrink: 0;
+  flex-grow: 1;
+  padding: 0 50px;
+}
+
+header,
+nav {
+  display: flex;
```
Comment (irenhh) [inline src/style.css:22]: display styles are better to write for each elements separately

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+nav {
+  align-items: center;
+}
+
+header > img {
```
Comment (irenhh) [inline src/style.css:29]: don't increase specificity, just add class name

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  margin-top: 10px;
+  width: 40px;
+  height: 40px;
+}
+
+a {
```
Comment (irenhh) [inline src/style.css:35]: class names

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  position: relative;
+  margin-right: 20px;
+  font-family: Roboto;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```
Comment (irenhh) [inline src/style.css:44]: where did you find such styles? 22.5, why not 22.3?

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  position: absolute;
+  z-index: 1;
```
Comment (irenhh) [inline src/style.css:62]: z-index is really necessary here?

File: src/style.css (hunk 6 above)
Comment (liliya-dev) [reply src/style.css:44]: https://prnt.sc/s559x0  here

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+  flex-grow: 1;
+  padding: 0 50px;
+  display: flex;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav {
```
Comment (irenhh) [inline src/style.css:24]: combine this with the prev selector, they are the same

File: src/style.css (hunk 6 above)
Comment (irenhh) [reply src/style.css:57]: let is be as it is for now

**Pull request closed without merging**